### ISSUE TITLE
When exception comes from non app code, don't display 'only app code'.

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry#8398]
#TLDR; display the callsite of error, even if 'not app code':

It is not usual but at times the error comes not from user code, but from the underlying framework or libraries:

Take for example:
![image](https://user-images.githubusercontent.com/1633368/39870775-d70413ac-5462-11e8-9992-d95ac1c09649.png)

In such case, the code which raised the error is hidden from the user. One needs to select _Full_ and then expand further:

![image](https://user-images.githubusercontent.com/1633368/39870822-fc8669e0-5462-11e8-8a15-5c43bdf57f15.png)

In this example, there's a bug in the .NET Framework version 4.7.2 so the exception is actually thrown from:
**System.Data.SqlClient.SqlConnection.CacheConnectionStringProperties** at line 27

**Comment by dcramer** [comment]
seems like a good idea imo

We should just make sure that the default expanded frame is still the 'first app frame'. I know we had/have a bug with this on certain views (chained exceptions) where that logic was wrong, and I wouldn't want us to make it more-wrong.

**Comment by bruno-garcia** [comment]
I think this is resolved. I recall seeing the grayed out smaller framework frame after the last inApp=true.
